server/util/live-loan/live-loan-draw.js

```diff
@@ -154,14 +186,180 @@ module.exports = async function draw(loanData) {
-		tracer.trace('canvasPool.recycle', () => canvasPool.recycle(canvas));
+		tracer.trace('legacyCanvasPool.recycle', () => legacyCanvasPool.recycle(canvas));
 
 		return buffer;
 	} catch (e) {
 		// Recycle canvas for use in other requests
 		if (canvas) {
-			tracer.trace('canvasPool.recycle', () => canvasPool.recycle(canvas));
+			tracer.trace('legacyCanvasPool.recycle', () => legacyCanvasPool.recycle(canvas));
 		}
 		throw e;
 	}
+}
+
+async function drawClassic(loanData) {
+	// Get canvas & context
+	const canvas = tracer.trace('classicCanvasPool.use', () => classicCanvasPool.use());
+	const ctx = tracer.trace('canvas.getContext', () => canvas.getContext('2d', { alpha: false }));
+
+	const borrowerImgMargin = 8 * classicResizeFactor;
+	const borrowerImgAspectRatio = 0.75;
+	const borrowerImgWidth = classicCardWidth - (2 * borrowerImgMargin);
+	const borrowerImgHeight = borrowerImgWidth * borrowerImgAspectRatio;
+
+	const kivaColors = {
+		action: '#2B7C5F',
+		brand: '#2AA967',
+		textPrimary: '#212121',
+		bgSecondary: '#f5f5f5',
+		bgTertiary: '#e0e0e0',
+		white: '#ffffff'
+	};
+
+	try {
+		// Canvas prep
+		tracer.trace('canvas-prep', () => {
+			ctx.textAlign = 'left';
+			ctx.textBaseline = 'top';
+			ctx.fillStyle = kivaColors.white;
+			ctx.fillRect(0, 0, classicCardWidth, classicCardHeight);
+		});
+
+		// Borrower Image
+		await tracer.trace('borrower-image', async () => {
+			const borrowerImg = await tracer.trace('loadImage', async () => loadImage(loanData.image.retina));
+			ctx.save();
+			// eslint-disable-next-line max-len
+			roundRect(ctx, borrowerImgMargin, borrowerImgMargin, borrowerImgWidth, borrowerImgHeight, 16 * classicResizeFactor);
+			ctx.clip();
+			ctx.drawImage(borrowerImg, borrowerImgMargin, borrowerImgMargin, borrowerImgWidth, borrowerImgHeight);
+			ctx.restore();
+		});
+
+		// Borrower country
+		tracer.trace('borrower-country', () => {
+			const countryXPos = 16 * classicResizeFactor;
+			const countryYPos = borrowerImgHeight - (22 * classicResizeFactor);
+			ctx.font = `500 ${14 * classicResizeFactor}px "Kiva Post Grot"`;
+			drawPill(
+				ctx,
+				loanData.geocode.country.name,
+				countryXPos,
+				countryYPos,
+				4 * classicResizeFactor,
+				kivaColors.textPrimary,
+				kivaColors.white,
+				mdiMapMarker,
+			);
+		});
+
+		// Borrower use
+		tracer.trace('borrower-use', () => {
+			const useXPos = 12 * classicResizeFactor;
+			const useYPos = (28 * classicResizeFactor) + borrowerImgHeight;
+			const useWidth = borrowerImgWidth - (8 * classicResizeFactor);
+			const useMaxLines = 2;
+			const useMaxLineHeight = 17 * 1.5 * classicResizeFactor;
+			const useText = getLoanUse({ ...loanData, version: 2 });
+			ctx.fillStyle = kivaColors.textPrimary;
+			ctx.font = `400 ${17 * classicResizeFactor}px "Kiva Post Grot"`;
+			wrapText(ctx, useText, useXPos, useYPos, useWidth, useMaxLines, useMaxLineHeight);
+		});
+
+		// Loan callouts
+		tracer.trace('loan-callouts', () => {
```

Comment:
Theoretically a loan with two loan callouts might have the pills cutoff as they extend outside the card, but it's so uncommon that I couldn't quickly find an example of a current loan with 2 long pills. Just something worth mentioning, since it was an edge case we discussed in the Vue app. Probably something that can be addresses if it ever comes up...

Reply:
Yeah, I think that's small enough of an edge case and such a minor visual issue that we can deal with it if it ever comes up and we decide it's a priority.